index.html

```diff
@@ -1599,15 +1600,10 @@ <h4>
               Output
             </dt>
             <dd>
-              A {{Promise}}
+              A new {{Promise}}
```

Comment:
Not sure adding "new" here is needed, be it only because if you add it here, then you may want to add it to other algorithms as well for consistency. For example, the `PresentationRequest` constructor algorithm always returns a "new" `PresentationRequest` but the output is "A `PresentationRequest` object".